internal/versions/versions.go

```diff
@@ -172,7 +173,11 @@ func (h *handler) getReleaseImageFromCache(openshiftVersion, cpuArchitecture str
 			return nil, err
 		}
 		releaseImage = funk.Find(releaseImages, func(releaseImage *models.ReleaseImage) bool {
-			return *releaseImage.OpenshiftVersion == versionKey
+			// Starting from OCP 4.12 multi-arch release images do not have "-multi" suffix
+			// reported by the CVO, but we still offer the sufix internally to allow for explicit
+			// selection or single- or multi-arch payload.
+			version := strings.TrimSuffix(*releaseImage.OpenshiftVersion, "-multi")
```

Comment:
Is this really the field we should be using to filter?

I know that's how it was before, but if this is intended to be a user-facing field then shouldn't we use `Version` instead which is the same for single and multi-arch?

Reply:
I think "that's how it was before" is the best answer here. If you asked me, I would be in favour of taking the time and rewriting the struct used to store the release image so that it does not hold `OpenshiftVersion` and `Version` fields which both don't have a well-defined usage (e.g. for some time I thought that only one of them is used by the backend and the other is exclusively for UI, but that's not true).

We should have only 1 version field. We should not have a field that artificially duplicates content reported by the CVO, because we have ability to extract this value ourselves. IMHO the simplest ReleaseImage type should hold only `URL` and `DisplayName`. Even architecture shouldn't be needed, because once you have the URL, you can get all the other info.

But those are big changes. It does not make our life easier that lots of them are only affecting UI which is not testable and you only see the effect once you manually rollout (because no CI will stop you from merging change that makes UI show silly stuff).